demos/security_barrier_camera_demo/README.md

```diff
@@ -96,7 +96,16 @@ To do inference for two video inputs using two asynchronous infer request on FPG
 ./security_barrier_camera_demo -i <path_to_video>/inputVideo_0.mp4 <path_to_video>/inputVideo_1.mp4 -m <path_to_model>/vehicle-license-plate-detection-barrier-0106.xml -m_va <path_to_model>/vehicle-attributes-recognition-barrier-0039.xml -m_lpr <path_to_model>/license-plate-recognition-barrier-0001.xml -d HETERO:FPGA,CPU -d_va HETERO:FPGA,CPU -d_lpr HETERO:FPGA,CPU -nireq 2
 ```
 
-> **NOTE**: For the `-tag` option (HDDL plugin only), you must specify the number of VPUs for each network in the `hddl_service.config` file located in the `<INSTALL_DIR>/deployment_tools/inference_engine/external/hddl/config/` direcrtory using the following tags:
+> **NOTE**: Some Optimization Hints with Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
+> The default scheduler strategy of security_barrier_camera_demo is Squeeze scheduler. To show the priority of Intel® Vision Accelerator Design with Intel® Movidius™ VPUs in this setting, you are suggested to config some parameters.
```

Comment:
I think, it should be clarified what `Squeeze scheduler` is (I don't know what it is, and other users too, I guess).
`in this setting, you are suggested to config some parameters.` - this does not seem to be clear in terms of the priority of Intel® Vision Accelerator Design with Intel® Movidius™ VPUs. `in this setting` - in setting of security_barrier_camera_demo? `you are suggested to config some parameters` - what parameters?

Reply:
Thank you for your reviewing, I will modify it again according to your suggestions and submit a new Pull Request.